core/src/main/java/org/apache/lucene/queryparser/classic/MissingFieldQueryExtension.java

```diff
@@ -33,7 +33,10 @@
 
     @Override
     public Query query(QueryParseContext parseContext, String queryText) {
-        return new ConstantScoreQuery(MissingQueryParser.newFilter(parseContext, queryText,
-                MissingQueryParser.DEFAULT_EXISTENCE_VALUE, MissingQueryParser.DEFAULT_NULL_VALUE, null));
+        Query query = MissingQueryParser.newFilter(parseContext, queryText, MissingQueryParser.DEFAULT_EXISTENCE_VALUE, MissingQueryParser.DEFAULT_NULL_VALUE);
```

Comment:
Is this because of your change or could newFilter() always return null even before the change?
Is it safe to return null if it does?